### ISSUE TITLE
Improve documentation on process for building tutorials

### ISSUE THREAD

**Issue by terichadbourne** [ProtoSchool/protoschool.github.io#178]
The current documentation at https://github.com/ProtoSchool/protoschool.github.io is out of date (given the addition of new types of lessons) and not thorough enough. 

Revised documentation (underway as part of PR #111) should: 
- [x] Explain Vue project structure
- [x] Make you feel confident you can build a tutorial without already knowing Vue
- [x] Explain the multiple options for lesson types (text only, code exercise, code exercise with file upload) and point to the templates for each Vue file
- [ ] Make it clear which parts of each template need to be updated
- [x] Explain what markdown files are needed and make a picture of where their content ends up in the rendered lesson
- [x] Explain how to make tutorial and all of its lessons appear on homepage and tutorials page (routing, etc.)
- [ ] Add guidance not just about how to use Vue to build a lesson but about how to structure an effective tutorial that adds an incremental piece of knowledge in each lesson 
- [ ] Highlight our goal to create beginner-friendly content and provide scaffolding
- [ ] Remind people to enter a new tutorial issue to make their content proposal and get feedback from Teri, get an outline approved, etc. before starting to build

@olizilla What am I missing above that would make the documentation effective? 

See current state of WIP draft at https://github.com/ProtoSchool/protoschool.github.io/blob/4d350c7ceb7c30bae80748217630b83071968bf6/README.md

**Comment by terichadbourne** [comment]
PR #244 adds screenshots but doesn't close this issue.

**Comment by terichadbourne** [comment]
Closing this out based on our Q2 work on documenting technical process. I've opened a new issue https://github.com/ProtoSchool/protoschool.github.io/issues/253 for adding more info on pedagogy and limitations of the system.
